.cirrus.yml

```diff
@@ -7,6 +7,16 @@ redhat_version_task:
         matrix:
           - image: fedora:35
           - image: fedora:34
+          - image: registry.access.redhat.com/ubi9-beta/ubi
+
+redhat8_version_task:
+    version_script:
+       - dnf -y install python3 python3-pip
+       - pip3 install "setuptools==40.6.3"
```

Comment:
It may not be explicit enough, but if you look at the previous task (redhat_version_task), its goal is to verify if Avocado will run "out of the box" with those systems.  Requiring packages from pip breaks the essence of those checks.

I'm not sure if we want to check (and support) those system packages (python3, python3-pip) plus pip packages (setuptools >= 40.6.3) approach.

Maybe, given CentOS 8 EOL, it's time to drop checks for EL 8 (keep them supported under LTS releases).